syz-manager/stats.go

```diff
@@ -35,6 +39,27 @@ type Stats struct {
 	haveHub    bool
 }
 
+
+func (mgr *Manager) initStats() {
+	mgr.stats.syz_execTotal = promauto.NewGauge(prometheus.GaugeOpts{
+                Name: "syz_execTotal",
```

Comment:
"Prometheus metrics and label names are written in snake_case":
https://prometheus.io/docs/instrumenting/writing_exporters/#naming